azext_edge/edge/util/az_client.py

```diff
@@ -183,6 +184,23 @@ def get_authz_client(subscription_id: str, **kwargs) -> "AuthorizationManagement
     )
 
 
+def get_keyvault_client(subscription_id: str, keyvault_name: str, **kwargs) -> "SecretClient":
+    from azure.keyvault.secrets import SecretClient
+
+    client = SecretClient(
+        credential=AZURE_CLI_CREDENTIAL,
+        subscription_id=subscription_id,
+        user_agent_policy=UserAgentPolicy(user_agent=USER_AGENT),
+        vault_url=f"https://{keyvault_name}.vault.azure.net",
+        **kwargs,
+    )
+
+    # wait to set the access token
+    sleep(5)
```

Comment:
for unit tests - make sure to make this become sleep(0) or mock it so they dont take 50 years

Reply:
yeah I already did that